src/modules/events-storage/events-storage.ts

```diff
@@ -0,0 +1,83 @@
+import { STORAGE_NAMES, STORAGE_TYPES } from "../storage-utils/storage-utils.types";
+import { StrigoEventsStorage, StrigoEvent } from "./events-storage.types";
+
+export function init(): StrigoEventsStorage {
+    try {
+        const currentEventsStorage = getEventsStorageData();
+        if (currentEventsStorage) {
+            console.log('Events storage already exists');
+            return currentEventsStorage;
+        }
+
+        const storageEvents = { events: [] };
+        window[STORAGE_TYPES.LOCAL_STORAGE].setItem(STORAGE_NAMES.STRIGO_EVENTS, JSON.stringify(storageEvents));
+        return storageEvents;
+    } catch (error) {
+        console.log(error);
```

Comment:
```suggestion
        logger.error('Strigo SDK init failed', error);
```
or something like that.

The problem is that we don't give any sign for the origin of the problem, and since the code is running "inside" customer's app, we must be way more descriptive